components/tikv_util/src/security.rs

```diff
@@ -137,6 +150,7 @@ impl SecurityManager {
         } else {
             let fetcher = Box::new(Fetcher {
                 cfg: self.cfg.clone(),
+                last_modified_time: Arc::new(Mutex::new(SystemTime::now())),
```

Comment:
How about using None instead of a specific time?

Reply:
Fix